src/tests/Test.js

```diff
@@ -70,7 +70,7 @@ class Test {
 
   async actual(data) {
     return this.postprocess(
-      data ? this.interaction().parse(data) : this.interaction().run()
+      data ? this.interaction().parse(data) : await this.interaction().run()
```

Comment:
This seems wrong, but it's confusingly written and overnested so it's hard to tell. The typing is all broken here because `interaction` is just throwing? Everything here seems confusing and broken. Adding the await just makes it more confusing.

If all you're doing is returning an `await` statement, you don't need to include the await. For example:
```
actual(data) { return promiseData() }
```
Is functionally equivalent to:
```
async actual(data) { return await promiseData() }
```

Reply:
I agree this is confusingly written. Probably doesn't need an await, will confirm. I think I was running into something and thought this might've fixed it but could've gotten my wires crossed. I'm not super familiar with this code (and it's really old/not clearly written pseudo-react) so I was just poking at stuff. Hopefully we can re-write this in TS soon.